app/views/api/payments/index.json.jbuilder

```diff
@@ -2,11 +2,11 @@
 
 # License: AGPL-3.0-or-later WITH WTO-AP-3.0-or-later
 # Full license explanation at https://github.com/houdiniproject/houdini/blob/main/LICENSE
-json.data @subtransaction_payments, partial: '/api/subtransaction_payments/subtransaction_payment',
-																																				as: :subtransaction_payment
+json.data @payments, partial: '/api/payments/subtransaction_payment',
```

Comment:
I don't think this should be changed, the directory should still be `/api/subtransaction_payments`

Reply:
Solved!